### PR TITLE
Refactor shorts python file virtual guest into a single file.

### PR DESCRIPTION
**Pull request by FernandoOjeda**
Refactor shorts python file virtual guest into a single file https://github.com/softlayer/softlayer.github.io/issues/200.

Files removed:
1. https://sldn.softlayer.com/python/monthtodate/
2. https://sldn.softlayer.com/python/getVerboseVsiDetails/
3. https://sldn.softlayer.com/python/getKvmDetails/
4. https://sldn.softlayer.com/python/view_closed_alarm.py/
5. https://sldn.softlayer.com/python/createvsiwithsecgroup/
6. https://sldn.softlayer.com/python/view_active_alarms.py/
7. https://sldn.softlayer.com/python/simplifiedOrder.py/
8. https://sldn.softlayer.com/python/set_tags_virtual_server.py/
9. https://sldn.softlayer.com/python/search_tag.py/
10. https://sldn.softlayer.com/python/retrieve_metadata.py/
11. https://sldn.softlayer.com/python/reload_current_os_configuration.py/
12. https://sldn.softlayer.com/python/reboot_default.py/
13. https://sldn.softlayer.com/python/get_virtual_guests_details.py/
14. https://sldn.softlayer.com/python/get_virtual_server_notes.py/

### CODE REVIEW

File: content/python/virtual_guest.py.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
+---
+title: "Working with Virtual Guest"
```
Comment (allmightyspiff) [inline content/python/virtual_guest.py.md:2]: I actually created a "Working with Virtual Guests" merged example already. https://github.com/softlayer/githubio_source/pull/367/files

Change the title to "More Virtual Guest Examples"

File: content/python/virtual_guest.py.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+description: "A few examples on interacting with Virtual Guest"
+date: "2020-07-09"
+classes: 
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Account"
+tags:
+    - "virtualservers"
```
Comment (allmightyspiff) [inline content/python/virtual_guest.py.md:9]: tags should be `vsi` and `virtualguest`

File: content/python/virtual_guest.py.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+This method is a simplified alternative to interacting with the ordering system directly.
+
+```python
+import SoftLayer
+import json
+
+USERNAME = 'set me'
+API_KEY = 'set me'
+
+client = SoftLayer.Client(username=USERNAME, api_key=API_KEY)
```
Comment (allmightyspiff) [inline content/python/virtual_guest.py.md:24]: For these examples, please remove the `USERNAME` and `API_KEY` bits. Idealls the `SoftLayer.Client()` should be reading from the config file, and makes the examples a bit easier to read in my opinion

File: content/python/virtual_guest.py.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+
+try:
+    response = virtualGuestService.createObject(order)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to place the order. faultCode=%s, faultString=%s" % (e.faultCode, e.faultString))
+
+```
+
+### Create a virtual server and attach Security Groups
```
Comment (allmightyspiff) [inline content/python/virtual_guest.py.md:60]: Change this example to create a VSI with a flavor, not CPU/Memory

File: content/python/virtual_guest.py.md [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+description: "A few examples on interacting with Virtual Guest"
+date: "2020-07-09"
+classes: 
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Account"
+tags:
+    - "virtualservers"
+---
+
+### Create Virtual Server
```
Comment (allmightyspiff) [inline content/python/virtual_guest.py.md:12]: Change this example to create a VSI with a flavor, not CPU/Memory

File: content/python/virtual_guest.py.md (hunk 1 above)
Comment (FernandoOjeda) [reply content/python/virtual_guest.py.md]: Done.

File: content/python/virtual_guest.py.md (hunk 5 above)
Comment (FernandoOjeda) [reply content/python/virtual_guest.py.md:13]: Done.

**Pull request merged**